package ru.test.findmaxnumber.api.exception;

public class InternalServerError extends Throwable {
    public InternalServerError(String message) {

    }

}
